packages/@aws-cdk/aws-events/package.json

```diff
@@ -84,6 +84,8 @@
   "awslint": {
     "exclude": [
       "from-method:@aws-cdk/aws-events.Rule",
+      "from-method:@aws-cdk/aws-events.CustomEventBus",
```

Comment:
The abstract class exports a `from` method which is why I added it to the `exclude`. Maybe this hierarchy isn't the best way to represent the two types of event buses?

Reply:
If the interface is called `IEventBus`, CDK users have been trained to look for the `from` method as `EventBus.fromXxx()`, so I would put it there. Since `CustomEventBus` and `PartnerEventBus` are more about the parameter used to initialize `EventBus`, I'm not sure that it's necessary to have them as classes.